### ISSUE TITLE
[Bug] new error with 1.14.4 

### ISSUE THREAD

**Issue by ThrashJazzAssassin** [jean-emmanuel/open-stage-control#773]
```
(ERROR, CLIENT) Uncaught TypeError: this.getProp(...).match is not a function
    at ownKeys (http://192.168.0.57:8000/client/open-stage-control-client.js?version=1.14.4:2528:3366)
    at widgetManager=require("./managers/widgets"),stateManager=require("./managers/state"),{deepCopy:deepCopy}=require("./utils"),Parser=class{constructor(){this.iterators={},this.widgets={},this.defaults={}}getIterator (http://192.168.0.57:8000/client/open-stage-control-client.js?version=1.14.4:2438:917)
    at s (http://192.168.0.57:8000/client/open-stage-control-client.js?version=1.14.4:2582:8893)
    at widgetManager=require("./managers/widgets"),stateManager=require("./managers/state"),{deepCopy:deepCopy}=require("./utils"),Parser=class{constructor(){this.iterators={},this.widgets={},this.defaults={}}getIterator (http://192.168.0.57:8000/client/open-stage-control-client.js?version=1.14.4:2438:917)
    at s (http://192.168.0.57:8000/client/open-stage-control-client.js?version=1.14.4:2582:8893)
    at widgetManager=require("./managers/widgets"),stateManager=require("./managers/state"),{deepCopy:deepCopy}=require("./utils"),Parser=class{constructor(){this.iterators={},this.widgets={},this.defaults={}}getIterator (http://192.168.0.57:8000/client/open-stage-control-client.js?version=1.14.4:2438:917)
    at s (http://192.168.0.57:8000/client/open-stage-control-client.js?version=1.14.4:2582:8893)
    at widgetManager=require("./managers/widgets"),stateManager=require("./managers/state"),{deepCopy:deepCopy}=require("./utils"),Parser=class{constructor(){this.iterators={},this.widgets={},this.defaults={}}getIterator (http://192.168.0.57:8000/client/open-stage-control-client.js?version=1.14.4:2438:917)
    at s (http://192.168.0.57:8000/client/open-stage-control-client.js?version=1.14.4:2582:8893)
    at widgetManager=require("./managers/widgets"),stateManager=require("./managers/state"),{deepCopy:deepCopy}=require("./utils"),Parser=class{constructor(){this.iterators={},this.widgets={},this.defaults={}}getIterator (http://192.168.0.57:8000/client/open-stage-control-client.js?version=1.14.4:2438:917)

``

**Comment by ThrashJazzAssassin** [comment]
[REAPER OpenStageControl TJA 0.2.3looper26.json.zip](https://github.com/jean-emmanuel/open-stage-control/files/8017929/REAPER.OpenStageControl.TJA.0.2.3looper26.json.zip)

**Comment by jean-emmanuel** [comment]
Fixed, v1.14.5 on the way
